src/components/card/card.component.tsx

```diff
@@ -65,18 +88,28 @@ const Card = ({
     }
   }, [ref]);
 
+  useLayoutEffect(() => {
```

Comment:
**question**: as this is a warning rather than a critical error, could we use a regular React effect here?

That way we wouldn't need to store `interactive` as an imperative value.

Reply:
Good point, I don't think we need the effect at all tbh